generators/app/templates/dotnet/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM <%= baseImageName %>
+COPY project.json /app/
+WORKDIR /app
+RUN ["dotnet", "restore"]
+COPY . /app
+RUN ["dotnet", "build", "-c", "<%= environment %>"]
+EXPOSE <%= portNumber %>
+ENTRYPOINT ["dotnet", "run"]
```

Comment:
Yes, these are just fro getting an image built and a container running.

---
In reply to: [64834074](https://github.com/Microsoft/generator-docker/pull/57#discussion_r64834074) [](ancestors = 64834074)